appengine-java8/tasks/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Cloud Tasks App Engine Queue Samples
+
+Sample command-line program for interacting with the Cloud Tasks API
+using App Engine queues.
+
+App Engine queues push tasks to an App Engine HTTP target. This directory
+contains both the App Engine app to deploy, as well as the snippets to run
+locally to push tasks to it, which could also be called on App Engine.
+
+`CreateTask.java` is a simple command-line program to create
+tasks to be pushed to the App Engine app.
+
+`TaskServlet.java` is the main App Engine app. This app serves as an endpoint to receive
+App Engine task attempts.
+
+
+## Initial Setup
+
+ * Set up a Google Cloud Project and enable billing.
+ * Enable the
+ [Cloud Tasks API](https://console.cloud.google.com/launcher/details/google/cloudtasks.googleapis.com).
+ * Download and install the [Cloud SDK](https://cloud.google.com/sdk).
+ * Download and install [Maven](http://maven.apache.org/install.html).
+
+## Creating a queue
+
+To create a queue using the Cloud SDK, use the following gcloud command:
+
+```
+gcloud beta tasks queues create-app-engine-queue my-appengine-queue
+```
+
+Note: A newly created queue will route to the default App Engine service and
+version unless configured to do otherwise.
+
+## Deploying the App Engine app
+[Using Maven and the App Engine Plugin](https://cloud.google.com/appengine/docs/flexible/java/using-maven)
+& [Maven Plugin Goals and Parameters](https://cloud.google.com/appengine/docs/flexible/java/maven-reference)
+
+### Running locally
+
+```
+mvn appengine:run
+```
+### Deploying
+
+```
+mvn appengine:deploy
+```
+
+Verify the index page is serving:
+
+```
+gcloud app browse
+```
+
+## Running the Samples
+
+Set environment variables:
+
+First, your project ID:
+
+```
+export PROJECT_ID=my-project-id
+```
+
+Then the queue ID, as specified at queue creation time. Queue IDs already
+created can be listed with `gcloud beta tasks queues list`.
+
+```
+export QUEUE_ID=my-appengine-queue
+```
+
+And finally the location ID, which can be discovered with
+`gcloud beta tasks queues describe $QUEUE_ID`, with the location embedded in
+the "name" value (for instance, if the name is
+"projects/my-project/locations/us-central1/queues/my-appengine-queue", then the
+location is "us-central1").
+
+```
+export LOCATION_ID=us-central1
+```
+
+Create a task, targeted at the `log_payload` endpoint, with a payload specified:
```

Comment:
Won't affect execution since the CreateTask class has it correct, but the endpoint should be 'tasks/create' here like you have below. Because it's important to be different from python.

Reply:
Done.